examples/NeuralNetwork-XOR/index.js

```diff
@@ -0,0 +1,53 @@
+let nn;
+let training_data = [{
+  inputs: [0, 0],
+  targets: [0]
+}, {
+  inputs: [1, 0],
+  targets: [1]
+}, {
+  inputs: [0, 1],
+  targets: [1]
+}, {
+  inputs: [1, 1],
+  targets: [0]
+}];
+
+// let lr_slider;
+
+function setup() {
+  createCanvas(200, 200);
+  nn = new ml5.NeuralNetwork(2, 4, 1);
+  // lr_slider = createSlider(0.01, 0.1, 0.05, 0.01);
+}
+
+function draw() {
+  background(0);
+
+  // nn.learningRate = lr_slider.value();
+
+  for (let i = 0; i < 1; i++) {
```

Comment:
is the loop necessary since you're only looping through the block once?

Reply:
It's not necessary! I left it at 1 for sorting out the code, but for the training to complete I've got to run more than one data point each time through draw.  I should probably close this pull request as I'm working on a new way to implement these "vanilla neural network" examples that I hope to do in April. 

The most recent code for this (without deeplearn.js/ml5) is here:

https://github.com/CodingTrain/Toy-Neural-Network-JS/tree/master/examples/xor

Reply:
cool, thanks!